evm_loader/lib/src/account_storage.rs

```diff
@@ -1270,6 +1271,13 @@ impl<T: Rpc> SyncedAccountStorage for EmulatorAccountStorage<'_, T> {
                 .map_err(map_neon_error)?
                 .borrow_mut();
 
+            if storage_data.owner != system_program::ID
+                && storage_data.is_empty()
```

Comment:
`(storage_data.owner != system_program::ID) && storage_data.is_empty()` is almost never true.
For non existing accounts: `storage_data.owner == system_program::ID` and `storage_data.is_empty()`
For existing: `storage_data.owner != system_program::ID` and in 99.99% cases `storage_data` is not empty.

Reply:
Removed "storage_data.owner != system_program::ID && storage_data.is_empty()"   [2c20d0e](https://github.com/neonlabsorg/neon-evm/pull/541/commits/0aaaecaca1edf4333751ed2de27b3b9f0ceabfa5) - tests failed 

Removed only ""storage_data.owner != system_program::ID"  484fb43e9ded3803e526bdcea70930ad5 - tests passed